shell.nix

```
@@ -1,17 +1,9 @@
+{ system ? builtins.currentSystem
+, config ? {}
+, localPkgs ? import ./. { inherit config system; }
+}:
 let
-  mainShell = (import ./release.nix).cardano-shell.env;
-  pkgs = import (import ./nixpkgs.nix) { config = import ./config.nix; };
-  runCoveralls = pkgs.stdenv.mkDerivation {
-    name = "run-coveralls";
-    buildInputs = [ pkgs.haskellPackages.stack-hpc-coveralls pkgs.stack ];
-    shellHook = ''
-      echo '~~~ stack test'
-      stack test --coverage
-      echo '~~~ shc'
-      shc --repo-token=$COVERALLS_REPO_TOKEN cardano-shell cardano-shell-test
-      exit
-    '';
-  };
+  mainShell = localPkgs.nix-tools.libs.cardano-shell.env;
```

Change this to:
```suggestion
  mainShell = localPkgs.nix-tools.libs.cardano-shell;
```
It should really be `pkgSet.config.hsPkgs.cardano-shell.components.all` though.

---

Neither of these does the trick, though.